cmd/podman/pod_inspect.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"encoding/json"
+
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/urfave/cli"
+)
+
+var (
+	podInspectFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "latest, l",
+			Usage: "act on the latest pod that podman is aware of",
+		},
+	}
+	podInspectDescription = "display the configuration for a pod by name or id"
+	podInspectCommand     = cli.Command{
+		Name:                   "inspect",
+		Usage:                  "displays a pod configuration",
+		Description:            podInspectDescription,
+		Flags:                  podInspectFlags,
+		Action:                 podInspectCmd,
+		UseShortOptionHandling: true,
+		ArgsUsage:              "[POD_NAME_OR_ID]",
+	}
+)
+
+func podInspectCmd(c *cli.Context) error {
+	var (
+		pod *libpod.Pod
+	)
+	args := c.Args()
+	runtime, err := libpodruntime.GetRuntime(c)
+	if err != nil {
+		return errors.Wrapf(err, "could not get runtime")
+	}
+	defer runtime.Shutdown(false)
+
```

Comment:
We should check if both args and latest were both passed. checkMutuallyExclusiveFlags(cli.context) does this in cmd/podman/utils.go

Reply:
done